src/schema/v2/__tests__/partner.test.js

```diff
@@ -218,108 +239,236 @@ describe("Partner type", () => {
                 },
-              },
-              {
-                node: {
-                  slug: "david-leventi-rezzonico",
+                {
+                  node: {
+                    slug: "david-leventi-rezzonico",
+                  },
                 },
-              },
-              {
-                node: {
-                  slug: "virginia-mak-hidden-nature-08",
+                {
+                  node: {
+                    slug: "virginia-mak-hidden-nature-08",
+                  },
                 },
-              },
-            ],
+              ],
+            },
           },
-        },
+        })
       })
-    })
 
-    it("returns hasNextPage=true when first is below total", async () => {
-      const query = `
-        {
-          partner(id:"bau-xi-gallery") {
-            artworksConnection(first:1) {
-              pageInfo {
-                hasNextPage
+      it("returns hasNextPage=true when first is below total", async () => {
+        const query = gql`
+          {
+            partner(id: "bau-xi-gallery") {
+              artworksConnection(first: 1) {
+                pageInfo {
+                  hasNextPage
+                }
               }
             }
           }
-        }
-      `
+        `
 
-      const data = await runQuery(query, context)
+        const data = await runQuery(query, context)
 
-      expect(data).toEqual({
-        partner: {
-          artworksConnection: {
-            pageInfo: {
-              hasNextPage: true,
+        expect(data).toEqual({
+          partner: {
+            artworksConnection: {
+              pageInfo: {
+                hasNextPage: true,
+              },
             },
           },
-        },
+        })
       })
-    })
 
-    it("returns hasNextPage=false when first is above total", async () => {
-      const query = `
-        {
-          partner(id:"bau-xi-gallery") {
-            artworksConnection(first:3) {
-              pageInfo {
-                hasNextPage
+      it("returns hasNextPage=false when first is above total", async () => {
+        const query = gql`
+          {
+            partner(id: "bau-xi-gallery") {
+              artworksConnection(first: 3) {
+                pageInfo {
+                  hasNextPage
+                }
               }
             }
           }
-        }
-      `
+        `
 
-      const data = await runQuery(query, context)
+        const data = await runQuery(query, context)
 
-      expect(data).toEqual({
-        partner: {
-          artworksConnection: {
-            pageInfo: {
-              hasNextPage: false,
+        expect(data).toEqual({
+          partner: {
+            artworksConnection: {
+              pageInfo: {
+                hasNextPage: false,
+              },
             },
           },
-        },
+        })
+      })
+
+      describe("when shallow is false", () => {
+        it("does not return deep artwork data", async () => {
```

Comment:
i think `"does not return deep artwork data"` is a little misleading here so i'd want to say that it does not call `partnerArtworksAllLoader` but it does here. I'm not sure how in the test to make sure the context doesn't have access to that loader. in real life, it wouldn't w/o the authentication

Reply:
(either way i think it's still good to still test that the data is returned properly so that we are making sure nothing is broken too)

Reply:
@annacarey wait, with `shallow: false`, we DO expect it to be calling `partnerArtworksAllLoader`, yes? Because it's fetching "deep" data?

Reply:
isn't our `it` just wrong here, and should instead read `it("returns deep artwork data", ...`

Reply:
Re: your original comment, I agree that `it("calls partnerArtworksCallLoader", ...` would be more descriptive of what we're actually testing

Reply:
i think no because it's under the `query is not authenticated` block here so our query should not have access to `partnerArtworksAllLoader` in the context. that is my understanding

Reply:
Ah ok, after reading this a few more times I think I get the problem. We're running the UNauthenticated query with `shallow: false`, so we expect `partnerArtworksAllLoader` to not be called. When we try this out in a console, what happens? is this the scenario where we get a full-on error back from Metaphysics, or does it just load "shallow" data?

Reply:
I'm not sure how to run an unauthenticated query in console?

Reply:
Closing the loop on this! I decided to specify the contexts for each of the describe blocks with the authenticated query vs. unauthenticated query. The unauthenticated queries does not have access to `partnerArtworksAllLoader` so the tests now reflect that change. Thanks for your help on this @mdole!